dateparser/__init__.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-__version__ = '0.6.0'
+__version__ = '0.7.0.dev1'
```

Comment:
Not sure about the procedures for pull requests, but I believe I have reverted as you proposed.
Thanks for the comment!